src/components/mention-area.component.js

```diff
@@ -1,36 +1,30 @@
 import React, { Component } from 'react';
 import fuzzysort from 'fuzzysort';
-import {
-  Text,
-  View,
-  TouchableOpacity,
-  Animated,
-  ScrollView,
-  StyleSheet,
-} from 'react-native';
+import styled from 'styled-components/native';
+import { TouchableOpacity, Animated, ScrollView } from 'react-native';
 
 import { animations, fonts, normalize } from 'config';
 
-const styles = StyleSheet.create({
-  suggestionsRowContainer: {
-    padding: 5,
-    flexDirection: 'row',
-    paddingRight: 15,
-    paddingBottom: 15,
-  },
-  userDetailsBox: {
-    flex: 1,
-    margin: 5,
-  },
-  displayNameText: {
-    fontSize: normalize(12),
-    ...fonts.fontPrimary,
-  },
-  realNameText: {
-    fontSize: normalize(12),
-    ...fonts.fontPrimarySemiBold,
-  },
-});
+const SuggestionsRowContainer = styled.View`
+  flex-direction: row;
+  padding-top: 5;
+  padding-left: 5;
+  padding-right: 15;
+  padding-bottom: 15;
+`;
+
+const UserDetailsBox = styled.View`
+  flex: 1;
+  margin-left: 5;
```

Comment:
If just use `margin: 5;`, will it work?

Reply:
@lex111  When I use ```margin: 5;```, I got the following error. Seperating margin into ```margin-left```, ```margin-right```, ```margin-top```, and, ```margin-buttom``` works but I have no idea why.
![screen shot 2560-10-22 at 20 35 57](https://user-images.githubusercontent.com/20456165/31862503-3ee93c32-b769-11e7-9efa-8b4e99daa5cc.png)

Reply:
@lex111 It is due to styled component's [shorthands properties restriction](https://www.styled-components.com/docs/basics#react-native).